### DISCUSSION TITLE
indicator_ipi-scope.gc missing "true" code (SDK 1.13.0 rc3)

### DISCUSSION THREAD

**Discussion by tompeeszter** [OP-TED/eForms-SDK#1070] [Q&A] [answered]
We have noticed the following issue with the codelist **indicator_ipi-scope.gc** in SDK 1.13.0 rc3:
Although it's an indicator codelist code "true" is missing. (Only code  "false" exists.)

Will it be fixed in the final version?
Thanks for your answer in advance!

**Comment by ferrakaop** [comment, marked as answer]
This is a deliberate choice because there are no IPI implementing acts yet - so the indicator that IPI applies can only be false.

Note that international-procurement-instrument-measure.gc only has one "empty" value as a placeholder, which will actually never be used.

We will need to update these codelists in a future SDK version once there is an IPI implementing act.  SDK 1.13 allows us to have the fields etc in place ready to be used.

See also https://code.europa.eu/eproc/eforms/docs/-/blob/main/guides/gde_002_ipi.md for some background.

**Comment by tompeeszter** [comment]
Thanks a lot for the rapid answer!
